nyxx_lavalink/lib/src/model/SearchPlatform.dart

```diff
@@ -0,0 +1,20 @@
+part of nyxx_lavalink;
+
+/// Search platforms supported by Lavalink
+enum SearchPlatform {
```

Comment:
Please use IEnum, to store a value

Reply:
With IEnum `_stringPlatform` const could be removed and IEnum could hold the value

Reply:
Done